common/context.go

```diff
@@ -125,6 +126,36 @@ func (ctx *Context) InitializeConfigFromFile(muFile string) error {
 	return ctx.InitializeConfig(bufio.NewReader(yamlFile))
 }
 
+// SubstituteEnvironmentVariable performns environment variable substitution according to Issue #209 (Dynamic Variables)
```

Comment:
Doesn't look like this is being called now?

Reply:
it isn't -- I'm working on that now.